pkg/registry/apps/statefulset/strategy.go

```diff
@@ -100,6 +132,8 @@ func (statefulSetStrategy) PrepareForUpdate(ctx context.Context, obj, old runtim
 	dropStatefulSetDisabledFields(newStatefulSet, oldStatefulSet)
 	pod.DropDisabledTemplateFields(&newStatefulSet.Spec.Template, &oldStatefulSet.Spec.Template)
 
+	dropStatefulSetDisabledFields(newStatefulSet, oldStatefulSet)
```

Comment:
so there was a bug where `StatefulSetMinReadySeconds` were not dropped on Updates? 😢

Reply:
Didn't fully understand, don't we already call `dropStatefulSetDisabledFields` [here](https://github.com/kubernetes/kubernetes/pull/82162/commits/6cba2e5b130de26ddaedcc64758ac32393353d16#diff-453cdbb00bec23e49e2da14c1989f01ef2705560867257fe2630c3f516ec6404R135) and we moved the functions around?

Reply:
is there any action required here @alculquicondor  @ravisantoshgudimetla  ?

Reply:
Please look at my comment ^? We're calling `dropStatefulSetDisabledFields` twice? We already called it [here](https://github.com/kubernetes/kubernetes/pull/82162/commits/0487c6b9043861d967161ef2ccfe43129120779c#diff-453cdbb00bec23e49e2da14c1989f01ef2705560867257fe2630c3f516ec6404L101). Why should we do it?

Reply:
fixed

Reply:
This still applies